onnx_tf/handlers/backend/sequence_construct.py

```diff
@@ -0,0 +1,26 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("SequenceConstruct")
+class SequenceConstruct(BackendHandler):
+
+  @classmethod
+  def version_11(cls, node, **kwargs):
+    # create an empty sequence first
+    tensor_dict = kwargs["tensor_dict"]
+    dtype = tensor_dict[node.inputs[0]].dtype
+    input_sequence = tf.ragged.constant([], dtype=dtype)
+
+    # insert tensors at the end of sequence
+    for i in range(len(node.inputs)):
+      input_tensor = tf.expand_dims(tensor_dict[node.inputs[i]], 0)
+      if input_sequence.shape[0] == 0:
+        output_seq = tf.RaggedTensor.from_tensor(input_tensor)
+      else:
+        output_seq = tf.concat([input_sequence, input_tensor], axis=0)
```

Comment:
Thanks for the view and suggestion. However tf.ragged.stack doesn't work here as it always returns rank R+1, as documented "A RaggedTensor with rank R+1. result.ragged_rank=1+max(axis, max(rt.ragged_rank for rt in values]))." Switching to tf.ragged.stack directly would cause a shape/rank check assertion in ragged_concat_ops.py, _ragged_stack_concat_helper. tf.concat in fact would go through the same code path in ragged_concat_ops.py and pass the rank check.

Reply:
It would also cause assertion false if we abandon the for loop and use such code as `tf.ragged.stack(node.inputs)`?

Reply:
It would cause problems when taking the tensor out in SequenceAt because tf.ragged.stack() automatically converts tf.Tensors into tf.RaggedTensors. It is much easier to keep the elements in a sequence (tf.RaggedTensor) as Tensors. Basically we use a RaggedTensor to represent an ONNX sequence without having to further convert the elements between Tensor and RaggedTensor. Hope this helps explain why I used the concat instead of ragged.stack.